relay-cabi/src/processing.rs

```diff
@@ -127,8 +246,16 @@ pub unsafe extern "C" fn relay_store_normalizer_normalize_event(
     };
     validate_transaction(&mut event, &tx_validation_config)?;
 
+    let is_renormalize = config.is_renormalize.unwrap_or(false);
+
     let normalization_config = NormalizationConfig {
+        project_id: config.project_id,
+        client: config.client.clone(),
+        protocol_version: config.protocol_version.clone(),
+        key_id: config.key_id.clone(),
+        grouping_config: config.grouping_config.clone(),
```

Comment:
Do these have to be clone?

Reply:
Yep, `NormalizationConfig` expects `String`s (and `serde_json::Value` for `grouping_config`). I'll look into the config to see if it needs to own the data.